src/submitter.py

```diff
@@ -115,6 +115,18 @@ def construct_job_template(filename: str) -> dict:
         "specificationVersion": "jobtemplate-2023-09",
         "name": filename,
         "parameterDefinitions": [
+            {
+                "name": "AdaptorScriptsDir",
+                "description": "Directory containing adaptor scripts.",
+                "userInterface": {
+                    "control": "HIDDEN"
+                },
+                "type": "PATH",
+                "objectType": "DIRECTORY",
+                "dataFlow": "IN",
+                "default": "adaptor",
```

Comment:
I like this approach! Just one clarification question, instead of copying over, we are copying over the script to `adaptor` directory and submitting the directory as part of job right?

Reply:
Yes, exactly!